kiali/kiali
# Update the config for the open telemetry collector

### Describe the change
Update the config for the open telemetry collector (For multi cluster)

### Steps to test the PR

`hack/run-integration-tests.sh --test-suite frontend-primary-remote --setup-only true`
And check if it has traces. 

cy should pass.

### Automation testing

N/A

### Issue reference
N/A
